ocaml/xapi/xapi_vm_helpers.ml

```diff
@@ -884,6 +884,9 @@ let vif_inclusive_range a b =
    may be further restricted. *)
 
 let allowed_VBD_devices_HVM            = vbd_inclusive_range true 0 3
```

Comment:
I think that `allowed_VBD_devices_HVM` is no longer used after this patch, and should therefore be removed.